README.md

```diff
@@ -16,57 +16,54 @@ The trie class is composed of the trie_t struct and its respective operations.
 The trie_t struct is composed of:
```

Comment:
In the interest of time I'm not going to read this README now. We'll do a README review closer to the end of the project, unless you'd like me to do it now.

Reply:
That's fine!